odxtools/diaglayer.py

```diff
@@ -751,15 +753,16 @@ def __str__(self) -> str:
 
 
 def read_diag_layer_container_from_odx(et_element, enable_candela_workarounds=True):
-    short_name = et_element.find("SHORT-NAME").text
+    short_name = et_element.findtext("SHORT-NAME")
+    assert short_name is not None
```

Comment:
Not really related to the PR, but please raise `OdxError` instead of assert

Reply:
good suggestion. Let's keep it in mind. (in order to keep the code churn at an acceptable level, we should probably introduce an `odx_assert()` function for that?)

Reply:
Yes